qbft/spectest/tests/controller/futuremsg/first_height_not_started.go

```diff
@@ -0,0 +1,23 @@
+package futuremsg
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// FirstHeightNotStarted tests a future message for special case (first height not started)
```

Comment:
this comment is confusing since this function doesn't test anything but creates a result that the actual test can be tested against.

Reply:
This function is called either by run or when jsons are created.

Reply:
should open an issue

Reply:
opened #202